config/initializers/cache_stats.rb

```diff
@@ -1,6 +1,6 @@
-ActiveSupport::Notifications.subscribe(/cache_.*\.active_support/) do |name, start, finish, arg1, arg2|
-  Rails.logger.debug name: name, start: start, finish: finish, arg1: arg1, arg2: arg2
-end
+# ActiveSupport::Notifications.subscribe(/cache_.*\.active_support/) do |name, start, finish, arg1, arg2|
+#   Rails.logger.debug name: name, start: start, finish: finish, arg1: arg1, arg2: arg2
+# end
```

Comment:
Was this a debugging change?

Reply:
It gets too noisy to have this enabled all the time in dev, but it is very useful for cache debugging and monitoring in dev.  I'll had some better comments or tooling around this to make that more clear.